CI failing

[`configure: error: Could not find fontconfig! You might be able to fix this by running 'sudo apt-get install libfontconfig1-dev'.`](https://github.com/jspecify/jdk/actions/runs/13188406795/job/36815915167)

It was working, well, [two months ago](https://github.com/jspecify/jdk/commit/39339b00f9fcf290db3d40537717c0bfe42e0884).

If the problem is just that we need to install another package, I assume that that's not hard.

--- a/.github/workflows/jspecify.yml
+++ b/.github/workflows/jspecify.yml
@@ -24,7 +24,8 @@ jobs:
       run: |
         sudo apt-get update
         sudo apt-get install --only-upgrade apt
-        sudo apt-get install gcc-10 g++-10 libxrandr-dev libxtst-dev libcups2-dev libasound2-dev libfontconfig1-dev
+        sudo apt-get install gcc-10 g++-10 libcups2-dev libasound2-dev libfontconfig1-dev \
+            libx11-dev libxext-dev libxrender-dev libxrandr-dev libxtst-dev libxt-dev
     - name: Install jtreg
       run: |
         wget https://builds.shipilev.net/jtreg/jtreg-7.5%2B1.zip -O /tmp/jtreg.zip

--- a/.github/workflows/jspecify.yml
+++ b/.github/workflows/jspecify.yml
@@ -24,7 +24,7 @@ jobs:
       run: |
         sudo apt-get update
         sudo apt-get install --only-upgrade apt
-        sudo apt-get install gcc-10 g++-10 libxrandr-dev libxtst-dev libcups2-dev libasound2-dev
+        sudo apt-get install gcc-10 g++-10 libxrandr-dev libxtst-dev libcups2-dev libasound2-dev libfontconfig1-dev
     - name: Install jtreg
       run: |
         wget https://builds.shipilev.net/jtreg/jtreg-7.5%2B1.zip -O /tmp/jtreg.zip

